generate.sh

```diff
@@ -7,9 +7,13 @@ function ask1() { local yn; echo -n "$* [Y/n] "; read -r -n 1 yn; [[ "$yn" =~ [Y
 for f in "$srcdir"/{,.}*.mustache; do
     target=$(basename "$f" .mustache)
     case "$target" in
-        *.opam)
-            shortname=$(grep -e "shortname:" meta.yml | sed -e "s/^shortname: //")
-            target=${target/coq/coq-$shortname}
+        coq.opam)
+            shortname=$(grep -e "^shortname:" meta.yml | sed -e 's/^shortname:\s\+//')
```

Comment:
BTW, this is likely not robust with respect to the full YAML syntax. Since we are using a mustache encoding, which knows how to parse a YAML file, it'd be interesting to piggy-back on this. The implementation I use (go-mustache) is pretty trivial regarding what command-line arguments it takes, so the only way of doing this would be to create a `/tmp/shortname.mustache` file.

Reply:
Good point, thanks @Zimmi48 !

I've just pushed a new version that addresses this.

Do you think that `mustache file.yml - <<<"$mustache"` is portable enough?
(I mean, with `-` as filename)

Otherwise one could also do:
`mustache file.yml <(echo "$mustache")`
with a bash process substitution